example/src/main/scala/com/avast/server/toolkit/example/Main.scala

```diff
@@ -3,17 +3,19 @@ package com.avast.server.toolkit.example
 import cats.effect.Resource
 import com.avast.server.toolkit.example.config.Configuration
 import com.avast.server.toolkit.pureconfig.PureConfigModule
+import com.github.ghik.silencer.silent
 import zio.interop.catz._
 import zio.{Task, ZIO}
 
 object Main extends CatsApp {
 
   def program: Resource[Task, Unit] = {
     for {
-      configuration <- Resource.liftF(PureConfigModule.makeOrRaise[Task, Configuration])
+      _ <- Resource.liftF(PureConfigModule.makeOrRaise[Task, Configuration])
     } yield ()
   }
 
+  @silent("dead code")
```

Comment:
Which peace of this code is dead? It shouldn't or am I wrong?

Reply:
false positive :(